src/Germany/README.md

```diff
@@ -101,6 +102,29 @@ var clientId = client.SuccessResult.Id;
 
 **Creation of a new Tss id**
 ```csharp
-var tss = await client.CreateTssAsync(accessToken, TssState.Initialized, description: "Creating a test TSS.");
+var tss = await client.CreateTssAsync(accessToken);
 var tssId = tss.SuccessResult.Id;
 ```
+
+After the creation of a new TSS, it is necessary to save the Admin PUK code that is returned in the response, the PUK code will be used later (for admin authentication).
```

Comment:
This one can be removed when it's in the code (And pls mention this in the test as a comment too, when you introduce that.)